scripts/test-check-packages.sh

```diff
@@ -18,6 +18,6 @@ for d in test/packages/*/; do
   (
     cd $d
     elastic-package check -v
-    elastic-package test -v
+    elastic-package test -v -r xUnit
```

Comment:
Do you think we may need multiple reporters enabled? Console + xUnit?

Reply:
I was thinking about this too. At the moment I can't think of a use case for it. It should be easy enough to implement if/when the need arises so I would be in favor of leaving this functionality out until we need it. WDYT?

Reply:
Absolutely fine, please postpone it.